src/protocolgame.cpp

```diff
@@ -2692,7 +2692,7 @@ void ProtocolGame::sendOutfitWindow()
 		}
 
 		protocolOutfits.emplace_back(outfit.name, outfit.lookType, addons);
-		if (protocolOutfits.size() == 100) { // Game client doesn't allow more than 100 outfits
+		if (protocolOutfits.size() == 255) { // FIXME: Game client currently doesn't allow more than 255 outfits
```

Comment:
I'm not sure what is the action item for the `FIXME` comment.
If the client doesn't support it, there's nothing we can do, right?

Reply:
actually I left it for future use, because once we support a higher client version it will need to be raised

Reply:
if (protocolOutfits.size() == std::numeric_limits<uint8_t>::max())
more descriptive why the limit is there :)